src/component/AgendaItem.jsx

```diff
@@ -1,114 +1,99 @@
-import React, { Component } from "react";
-import { connect } from "react-redux";
-import { Link } from "react-router-dom";
-import agenda_item_received from "../actions/Form";
-import { requestAgendas } from "../ducks/agendas";
-import { bindActionCreators } from "redux";
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { agendaItemReceived } from '../actions/Form';
+import { requestAgendas } from '../ducks/agendas';
+import { bindActionCreators } from 'redux';
 import { setHours, setMinutes } from 'date-fns';
-import format from "date-fns/format";
-//I can just get the id from the param and use to to fetch from Application state's agenda agendaitems
-//Now I just need to design a page and put informations on them
-import { Button, Card, Container, Loader } from "semantic-ui-react";
+import format from 'date-fns/format';
+// I can just get the id from the param and use to to fetch from Application state's agenda agendaitems
+// Now I just need to design a page and put informations on them
+import { Button, Card, Container, Loader } from 'semantic-ui-react';
+
+const recommendationReducer = (acc, curr) => `${acc}<br />${curr}`;
+const summaryReducer = (acc, curr) => {
+  if (!this.gotBackground) {
```

Comment:
Receiving "Uncaught TypeError: Cannot read property 'gotBackground' of undefined". Possibly update to `if (this && !this.hasOwnProperty('gotBackground')) {` as well as in AgendaFeedContainer.jsx

Reply:
Yeah, that was a stupid mistake... let me correct that (I have to put the function back in the class scope)